Tools/autotest/quadplane.py

```diff
@@ -2164,6 +2164,90 @@ def WindEstimateConsistency(self):
                 self.progress("Wind estimates correlated")
                 break
 
+    def FullRateClimbout(self):
+        '''test the full rate climbout after VTOL Takeoff functions'''
+        # Set from examining the autotest with enable/disable flag.
+        ALTITUDE_CUTOFF_M = 35
+        CRUISE_ALTITUDE_CUTOFF = 120
+        WAIT_AFTER_CRUISE_DELAY_S = 10
+
+        # Mission planning items.
+        CRUISE_WP_NUM = 2
+
+        self.progress("Testing full rate climbout.")
+
+        def run_test(flag):
+            filename = "mission.txt"
+            self.progress("Testing %s with flag %s" % (filename, bool(flag)))
+            self.progress("Setting parameters.")
+
+            self.set_parameter("Q_FR_CLMBOUT", int(flag))
```

Comment:
Technically a float for ArduPilot - at least for transport to the autopilot.



In either case, this cast isn't required